index.html

```diff
@@ -209,5 +209,81 @@ <h2 class="title"> Skills</h2>
 
     </section>
 
+    <section class="contact">
+        <article class="art-contact">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </article>
+
+        <form action="https://formspree.io/f/mgebgoge">
+            <input type="text" 
+            id="user_name" 
+            placeholder="Full Name"
+            maxlength="30"
+            required
+            >
+
+            <input type="email" 
+            id="user_email" 
+            placeholder="Email address">
+        
+            <textarea name="" id="textarea" 
+            cols="30" 
+            rows="10" 
+            placeholder="Write me something..."
+            maxlength="500"
+            required></textarea>
+
+            <div class="btn-div">
+                <button class="btn-contact" type="submit">
+                    Get in touch
+                </button>
+            </div>
+            
+        </form>
```

Comment:
- There are some issues for this reason the form is not submitted. Kindly fix those issues.

- [ ] Add post method to `form`. Such as: `<form action="https://formspree.io/f/mgebgoge" method="Post">`
- [ ] Add `name` attribute for the first input. Such as: `            <input type="text"  id="user_name"  name="full_name" placeholder="Full Name" maxlength="30" required > `
- [ ] Add `name="email`, `maxlength` and `required` to the email input field.
- [ ] Add `name=message/anything` to the text field.

***Kindly fix those issues and check before submitting the form is submitted or not.***

- After clicking `Get In Touch` button, you will see this kind of message:
![image](https://github.com/ulises2607/Portfolio/assets/60083437/648fce0f-10b5-4435-ba31-2c34a1389f33)